python/llm/portable-installer/setup.bat

```diff
@@ -0,0 +1,20 @@
+:: download python and extract zip
+powershell -Command "Start-BitsTransfer -Source https://www.python.org/ftp/python/3.9.13/python-3.9.13-embed-amd64.zip -Destination python-3.9.13-embed-amd64.zip"
+powershell -Command "Expand-Archive .\python-3.9.13-embed-amd64.zip -DestinationPath .\python-embed"
+del .\python-3.9.13-embed-amd64.zip
+
+set "python-embed=.\python-embed\python.exe"
+
+:: download get-pip.py and install
+powershell -Command "Invoke-WebRequest https://bootstrap.pypa.io/get-pip.py -OutFile .\python-embed\get-pip.py"
+%python-embed% .\python-embed\get-pip.py
+
+:: enable run site.main() automatically
+cd .\python-embed
+set "search=#import site"
+set "replace=import site"
+powershell -Command "(gc python39._pth) -replace '%search%', '%replace%' | Out-File -encoding ASCII python39._pth"
+cd ..
+
+:: install pip packages
+%python-embed% -m pip install bigdl-llm[all] transformers_stream_generator
```

Comment:
Maybe also add dependency: `tiktoken` and `einops`.

`einops` is required for mpt, falcon and qwen;
`tiktoken` is required for qwen

Reply:
I think we'd better separate the setup scripts/README and user scripts&README. We use setup.bat to build the zip, and the zip should only contain the user scripts and user guide.

Reply:
In user guide please add a step-by-step usage guide with screen snapshots.

Reply:
Also, you can upload a gif of the process in github to show the entire process.

Reply:
Already done as you said  :)